src/main/java/uk/gov/ons/ctp/response/action/scheduled/distribution/ActionDistributor.java

```diff
@@ -22,57 +19,57 @@
 import uk.gov.ons.ctp.response.action.service.ActionProcessingService;
 import uk.gov.ons.ctp.response.sample.representation.SampleUnitDTO;
 
-/**
- * This is the 'service' class that distributes actions to downstream services, ie services outside
- * of Response Management (ActionExporterSvc, NotifyGW, etc.).
- *
- * <p>This class has a self scheduled method wakeUp(), which looks for Actions in SUBMITTED state to
- * send to downstream handlers. On each wake cycle, it fetches the first n actions of each type, by
- * createddatatime, and forwards them to ActionProcessingService.
- */
+/** This is the service class that distributes actions to downstream services */
 @Component
 class ActionDistributor {
   private static final Logger log = LoggerFactory.getLogger(ActionDistributor.class);
 
-  @Autowired private AppConfig appConfig;
+  private AppConfig appConfig;
 
-  @Autowired private ActionRepository actionRepo;
+  private ActionRepository actionRepo;
+  private ActionCaseRepository actionCaseRepo;
+  private ActionTypeRepository actionTypeRepo;
```

Comment:
make final

Reply:
Making this final was having strange effects on the tests. It seemed a bit challenging to fix the tests around it so leaving the `final` out for now